services/compute/compute.go

```diff
@@ -666,6 +666,8 @@ type GalleryImageIdentifier struct {
 	Offer *string `json:"offer,omitempty"`
 	// Sku - The name of the gallery Image Definition SKU.
 	Sku *string `json:"sku,omitempty"`
+	// Version - Specifies the version of the platform image or marketplace image used to create the virtual machine.
+	Version *string `json:"version,omitempty"`
```

Comment:
This property doesnt exist in Azure sdk. Why are we introducing it here
https://github.com/Azure/azure-sdk-for-go/blob/main/services/compute/mgmt/2021-11-01/compute/models.go#L7900

Reply:
The Version is specific to http download from market place. 
Check how we do this using SFSImageProperties. Either use httpImageProperties or define a new struct and use it the same way as SfsImageProperties and pass it along as a string.

Try not to introduce params that would be specific to a type

Reply:
And Please document that changes in the design md